packages/api/src/webhooks/cannon.ts

```diff
@@ -69,7 +77,7 @@ export default class WebhookCannon {
     event.lastInterval = this.calcBackoff(event.lastInterval);
     event.status = "pending";
     event.retries = event.retries ? event.retries + 1 : 1;
```

Comment:
Since the `event` object is shared between different webhook deliveries, might be better to make a copy here instead
```suggestion
    event = {
      ...event,
      lastInterval: this.calcBackoff(event.lastInterval),
      status: "pending",
      retries: event.retries ? event.retries + 1 : 1,
    }
```

Reply:
Additionally, do we/should we have a limit of retries? Like not retry for more than 24 hours the same event.